pkgs/tools/misc/byobu/default.nix

```diff
@@ -1,5 +1,10 @@
-{ stdenv, fetchurl, python3, perl, textual-window-manager }:
+{ stdenv, fetchurl, makeWrapper, ncurses, python3, perl, textual-window-manager,
+  gettext, vim, bc, screen }:
 
+let
+  pythonEnv = python3.withPackages(ps: [ps.snack]);
+  binPath = stdenv.lib.makeBinPath [ screen textual-window-manager ];
```

Comment:
usage of this is commented out. is it needed?

Reply:
Thanks I removed it